src/backend/metal/src/device.rs

```diff
@@ -1366,25 +1414,15 @@ impl hal::Device<Backend> for Device {
     }
 
     fn create_framebuffer<I>(
-        &self, renderpass: &n::RenderPass, attachments: I, extent: image::Extent
+        &self, _render_pass: &n::RenderPass, attachments: I, extent: image::Extent
     ) -> Result<n::Framebuffer, FramebufferError>
     where
         I: IntoIterator,
         I::Item: Borrow<n::ImageView>
     {
-        let descriptor = metal::RenderPassDescriptor::new().to_owned();
-        descriptor.set_visibility_result_buffer(Some(&self.shared.visibility.buffer));
-        descriptor.set_render_target_array_length(extent.depth as NSUInteger);
-
-        let mut inner = n::FramebufferInner {
-            extent,
-            aspects: format::Aspects::empty(),
-            colors: SmallVec::new(),
-            depth_stencil: None,
-        };
-
+        /*
```

Comment:
Should this block just be removed entirely, or are we planning to revisit it soon?